redis/RedisSentinel.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * Helper autocomplete for php redis sentinel extension
+ *
```

Comment:
Please, add MIT License mention in the beginning of the file

Reply:
Done.